cmd/core/cmd/registry_windows.go

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2024, Cogent Core. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package cmd
+
+import (
+	"fmt"
+	"strings"
+
+	"cogentcore.org/core/base/errors"
+	"golang.org/x/sys/windows/registry"
+)
+
+// WindowsRegistryAddPath adds given path to the system path in the
+// windows registry.  Path string must be properly escaped when passed
+// as a double-quote string, e.g., "C:\\w64devkit\\bin".
```

Comment:
This is obvious and true for all Go strings.

Reply:
I had to look it up (of course).  what is obvious to you may not be to others..

Reply:
Just use backticks and don't document it. It would be ridiculous to have every function document how to specify the builtin types. `This function takes a 32-bit signed integer value. You can pass this as a variable or untyped integer constant, which can be expressed using decimal (ex: 4), hexadecimal (ex: 0xf), or octal (ex: 0o777) notation`. If you don't remember something about the language, it is incumbent upon you to figure that out, not the documentation. Again, use backticks and don't document it.